### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by bruk19**
1. Added html.index for desktop file
2. Added Css.style using media queries for desktop design 
3. Added all important icons
4. Added all important images
5. Added linters checker

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,176 +15,247 @@
...
-                    <h3>Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry's standard</p>
-                        <ul>
-                            <li class="list-btn">html</li>
-                            <li class="list-boot">bootstrap</li>
-                            <li class="list-btn">Ruby</li>
+                <div class="img">
+                    <img src="img/placeholder-new.png" alt="">
+                    <div class="my-workr">
```
Comment (roshan-bajgain) [inline index.html:62]: - [ ] If you see the Figma design then the image takes 60% and content takes 40% So kindly Implement it as Figma.
### your design 
his
### Figma design
figma

File: index.html [hunk 2]
```diff
@@ -15,176 +15,247 @@
...
+                <div class="dash-mob">
+                    <h2 class="mob-tit">Profesional Art Printing Data</h2> 
+                    <p class="mob-dics">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="mob-li">
+                        <li>HTML</li>
+                        <li>BootStrap</li>
+                        <li>Rubby</li>
+                    </ul>
+                    <button type="button">See Project</button>
                 </div>
```
Comment (roshan-bajgain) [inline index.html:105]: - [ ] There's a See Project button that takes you to the Live Demo of the featured project or to the GitHub repository. So kindly implement it.

File: index.html [hunk 3]
```diff
@@ -15,176 +15,247 @@
...
+        <div class="cont">
+            <div class="contact">
+                <p class="cont-dics">I'm always interested in hearing about new projects, 
+                    so if you'd like to chat please get in touch.</p>
+                <form class="ip-type" action="https://formspree.io/f/xzbokgee" method="post">
+                    <input type="text" class="input" placeholder="   Full Name" required>
+                    <input type="email" class="input" placeholder="   Email Address" required>
+                    <textarea name="msgt1" class="input-one" cols="30" rows="10" maxlength="500" placeholder="  Enter text here" required></textarea>
+                    <button type="button" class="cont-btn">Get in touch</button>
+                </form>
```
Comment (roshan-bajgain) [inline index.html:246]: - [ ] There should be one input for Full name this should take the space of both First Name and Last Name inputs (shown in the desktop version). Kindly implement it
- [ ] Kindly decrease the right and left padding of `Get in touch` button to make similar with Figma design
### your design
contact
### Figma design
figmacontact

File: index.html [hunk 4]
```diff
@@ -15,176 +15,247 @@
...
+        <div class="lin-la">
+            <hr class="lin">
+        </div>
+        <div class="last-icon">
+            <a href="#"><i class="fa-brands fa-github"></i></a>
+            <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+            <a href="#"><i class="fa-brands fa-angellist"></i></a>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#"><i class="fa-solid fa-m"></i></a>
+        </div>
```
Comment (roshan-bajgain) [inline index.html:258]: - [ ] Kindly decrease the `font-size` of icons to make similar to the Figma design
### your design
his
### Figma design
icons

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far! 😎 
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [ ] I would suggest you implement responsiveness on mobile devices as it is not responsive as expected. It is scrolling horizontally.
### video

https://user-images.githubusercontent.com/81727988/166094625-e5dd6499-913b-435d-88c7-b6f62c966452.mp4

_Check the comments under the review._

Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -445,29 +508,60 @@ hr {
...
   margin-top: 15px;
+  background-color: #f55800;
+  color: #fff;
+  height: 48px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  border: none;
+}
```
Comment (codepantha) [inline style.css]: - [ ] You have adjusted the padding of this button like the previous reviewer suggested, but it still doesn't look quite right. Please give this button a width of `131px` to be exactly as specified in the Figma design 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @bruk19 👋, 

## STATUS: REQUESTED CHANGES ♻️ 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As the previous reviewer suggested, please fix the horizontal scrolling on the mobile view 🙏
### Your Design
image
### Figma Design
image
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @bruk19  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Good job implementing most of the required changes 👍 
- Linter checks passing ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- Thank you for fixing the horizontal scrolling issue but with that came a white space in your mobile view. When you get time please fix that.
### Your mobile view
image

**Pull request merged**